### PR TITLE
checkpoint: add the empty-ns option

### PR DESCRIPTION
**Pull request by avagin**
For example:
    ./runc checkpoint --empty-ns network CTID
    
In this case criu creates a network namespace, but doesn't restore it.
    
We are going to use this option to restore docker containers and
Docker sets a hook to restore a network namespace.
    
https://github.com/xemul/criu/issues/165

Cc: @boucher

### CODE REVIEW

File: checkpoint.go [hunk 1]
```diff
@@ -88,3 +94,22 @@ func setManageCgroupsMode(context *cli.Context, options *libcontainer.CriuOpts)
...
+	specs.NetworkNamespace: syscall.CLONE_NEWNET,
+}
+
+func setEmptyNsMask(context *cli.Context, options *libcontainer.CriuOpts) error {
+	var nsmask int
+
+	for _, ns := range context.StringSlice("empty-ns") {
+		f, exists := namespaceMapping[specs.NamespaceType(ns)]
+		if !exists {
+			return fmt.Errorf("namespace %q does not supported", ns)
```
Comment (mrunalp) [inline checkpoint.go]: s/does/is

**Comment by mrunalp** [comment]
LGTM besides the one nit.

**Comment by boucher** [comment]
looks good to me too

**Comment by hqhq** [comment]
LGTM

**Pull request merged**